src/ServiceControl/App_Packages/StackTraceParser/StackTraceParser.cs

```diff
@@ -43,7 +43,9 @@ partial class StackTraceParser
                 ( " + Space + @"+
                     ( # Microsoft .NET stack traces
                     \w+ " + Space + @"+
-                    (?<file> [a-z] \: .+? )
+                    (?<file> ( [a-z] \: # Windows rooted path starting with a drive letter
```

Comment:
How about making this also a PR against the original source? This is a source package and should not be modified directly unless the original project is abandoned. Then we would need to detach the source package and have the sources here xopied

Reply:
Fix is already in the downstream, but downstream is using the old csproj format, so we can't use in in SC now. 

Just adding this for anybody reading this at a later date.